.github/workflows/release.yaml

```diff
@@ -0,0 +1,25 @@
+name: Release charm to other tracks and channels
```

Comment:
No need to separate it out now, but in future this feels like `release.yaml` could be a different PR as it does not depend on `publish.yaml`.  Would make it easier to approve and merge :)